src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,30 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.academy.dao");
 
+    public static void main(String[] args) {
+        Book book1 = new Book();
+        book1.setTitle("book1");
+        book1.setPrice(BigDecimal.valueOf(200));
+        Book book2 = new Book();
+        book2.setTitle("book2");
+        book2.setPrice(BigDecimal.valueOf(300));;
```

Comment:
There is an unnecessary semicolon at the end of this line. Each statement in Java should end with only one semicolon.